skyllh/core/signal_generator.py

```diff
@@ -390,6 +393,8 @@ def __init__(self, src_hypo_group_manager, dataset_list, data_list):
         data_list : list of DatasetData instances
             The list of DatasetData instances holding the actual data of each
             dataset. The order must match the order of ``dataset_list``.
+        kwargs
+            A typical keyword argument is the instance of MultiDatasetTCLLHRatio.
         """
         super(MultiSourceSignalGenerator, self).__init__(
             src_hypo_group_manager, dataset_list, data_list)
```

Comment:
Pass kwargs as we already added them as an argument.
```suggestion
            src_hypo_group_manager, dataset_list, data_list, **kwargs)
```